docs/vgrep.1.md

```diff
@@ -0,0 +1,63 @@
+
+`vgrep` runs on Linux, Windows and Mac OS.
+
+Note: `vgrep` is used to perform textual searches. On a technical level, vgrep serves as a front-end to grep or git-grep when invoking vgrep inside a git tree and uses `less` for displaying the results. All non-vgrep flags and arguments will be passed down to grep. Results of the last search are cached, so running vgrep without a new query will load previous results and operate on them.
+
+By default, the output will be written to less to make browsing large amounts of data more comfortable. vgrep --no-less will write to stdout.
+
+# Opening Matches
+
+vgrep can open the indexed file locations in an editor specified by the `EDITOR` environment variable. Opening one of the file locations from the previous example may look as follows:
+
+```
+# export EDITOR=gedit
+# vgrep --show 4
+```
+
+The default editor of vgrep is `vim` with the default flag to open a file at a specific line being `+` followed by the line number. If your editor of choice hits the rare case of a different syntax, use the `EDITORLINEFLAG` environment variable to adjust. For example, a `kate` user may set the environment to `EDITOR="kate"` and `EDITORLINEFLAG="-l"`.
+
+Note that `vgrep` does not allow for searching and opening files at the same time. For instance, `vgrep --show=files text` should be split in two commands: `vgrep text` and `vgrep --show=files`.
+
+# Interactive Shell
+
+Once vgreped, you can perform certain operations on the results such as limiting the range of indexed matches, listing matching files and directories and more.
+
+```
+Enter a vgrep command: ?
+vgrep command help: command[context lines] [selectors]
+         selectors: '3' (single), '1,2,6' (multi), '1-8' (range), 'all'
+          commands: print, show, context, tree, delete, keep, refine, files, grep, quit, ?
+```
+
+vgrep commands can be passed directly to the `--show/-s` flag, for instance as `--show c5 1-10` to show the five context lines of the first ten matched lines. Furthermore, the commands can be executed in an interactive shell via the `--interactive/-i` flag. Running `vgrep --interactive` will enter the shell directly, `vgrep --show 1 --interactive` will first open the first matched line in the editor and enter the interactive shell after.
+
+## COMMANDS
+
+| Command | Description                                                                                                                                                                                                                                                                           |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Please delete this line. Otherwise, it does not render correctly in smaller terminal windows.

Reply:
But that would remove the table, do you want it to be listed as bullet points?
I tested it on a small terminal window and it's rendered fine.
![Capture](https://user-images.githubusercontent.com/20889958/97125181-bd37eb80-173b-11eb-81f5-76abf2f8aeba.PNG)